src/app/about/page.js

```diff
@@ -1,25 +1,51 @@
+"use client";
 import React from "react";
 import Title from "@/components/Title";
```

Comment:
you need to implement the animation at the component level, not at the page level